docs/BackupNoSQL.md

```diff
@@ -0,0 +1,153 @@
+# Backup NoSQL database guide
+
+If you use a NoSQL database or multiple databases, you must **pause** the Scalar products to create a transactionally-consistent backup. This guide explains concrete steps of how to create the transactionally-consistent backup of managed databases with the Scalar products on a Kubernetes environment. Please refer to the following document if you want to know more details on the backup way of Scalar products.
+
+* [A Guide on How to Backup and Restore Databases Used Through ScalarDB](https://github.com/scalar-labs/scalardb/blob/master/docs/backup-restore.md)
+
+In this guide, we assume that using the automatic backup and PITR feature. So, we create a **period** where there are no ongoing transactions for restoration. You can restore data using PITR to the point-in-time in this **period**. Otherwise (if you restore data to point-in-time out of this **period**), the restored data becomes inconsistent data and Scalar products don't work properly.
+
+## Backup operations (create the **period** for restoration)
```

Comment:
We will create a new tool for creating the restoration period to cover the rare case issues.
However, it takes a bit of time to create, so I created this backup document based on the old (current) way at this time.
I will update this document after the new tool will be released.